packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/edit/logs-inputs-add-edit.controller.js

```diff
@@ -95,9 +96,14 @@ class LogsInputsAddEditCtrl {
     return this.inputAddEdit
       .load()
       .then((successData) => {
-        this.gotToNextStep(this.inputId || successData[0].item.inputId);
+        if (successData[0].item) {
+          return this.gotToNextStep(
+            this.inputId || successData[0].item.inputId,
+          );
+        }
+        return this.$q.reject();
```

Comment:
This will have no impact, what are you trying to do here ?

Reply:
Same answer.